File: common/src/main/java/io/github/candycalc/Phlexiful.java
package io.github.candycalc;

import io.github.candycalc.registry.BlockRegistry;
import io.github.candycalc.registry.EnchantRegistry;
import io.github.candycalc.registry.ItemRegistry;
import io.github.candycalc.registry.PatternRegistry;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public final class Phlexiful {
    public static final String MOD_ID = "phlexiful";
    public static final Logger LOGGER = LoggerFactory.getLogger("Phlexiful");
    public static void log(String message) {
        LOGGER.info(message);
    }

    public static void init() {
        PatternRegistry.register();
        BlockRegistry.register();
        ItemRegistry.register();
        EnchantRegistry.register();
    }
}

File: src/main/kotlin/io/github/candycalc/Phlexiful.kt
package io.github.candycalc

import io.github.candycalc.registry.BlockRegistry
import io.github.candycalc.registry.CreativeGroupRegistry
import io.github.candycalc.registry.ItemRegistry
import io.github.candycalc.registry.PatternRegistry
import net.fabricmc.api.ModInitializer
import org.slf4j.LoggerFactory

object Phlexiful : ModInitializer {
    val logger = LoggerFactory.getLogger("phlexiful")
	const val MOD_ID = "phlexiful"

	override fun onInitialize() {
		PatternRegistry.register()
		BlockRegistry.register()
		ItemRegistry.register()
		CreativeGroupRegistry.register()
	}
}